### PR TITLE
Align kernel module rules  with DISA STIG Benchmarks

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Better align the `kernel_module_disabled` template with DISA STIG Benchmarks
  - In SLE products, don't check for `install` keyword any more
  - In OL and RHEL products, add a check for `blacklist` keyword
- Stop checking `/etc/modprobe.conf` in RHEL, it was deprecated.

#### Rationale:

- Improve alignment of OL and RHEL kernel modules disabled rules with DISA Benchmark

### CODE REVIEW

**Comment by yuumasato** [comment]
@teacup-on-rockingchair @brett060102 @anivan-suse Hi, are you okay with https://github.com/ComplianceAsCode/content/pull/8942/commits/0b67d78f72dbaf90c398290e870e1d1bf2ca9d1a

It removes the checks for `install {{{ KERNMODULE }}} (/bin/false|/bin/true}`, the SLE contents only remediate for `blacklist {{{ KERNMODULE }}}`.

File: shared/templates/kernel_module_disabled/kubernetes.template [hunk 1]
```diff
@@ -13,7 +13,7 @@ spec:
     storage:
       files:
       - contents:
-          source: data:,install%20{{{ KERNMODULE | urlencode }}}%20/bin/true%0A
+          source: data:,install%20{{{ KERNMODULE | urlencode }}}%20/bin/true%0Ablacklist%20{{{ KERNMODULE }}}%0A
```
Comment (yuumasato) [inline shared/templates/kernel_module_disabled/kubernetes.template:16]: @jhrozek @Vincent056 Hi, does this change look appropriate?

**Review by jan-cerny — COMMENTED** [review]
looks great!

**Review by JAORMX — APPROVED** [review]
LGTM

**Review by JAORMX — APPROVED** [review]

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**